sigh/lib/sigh/download_all.rb

```diff
@@ -7,7 +7,7 @@ def download_all
       Spaceship.select_team
       UI.message "Successfully logged in"
 
-      Spaceship.provisioning_profile.all.each do |profile|
+      Spaceship.provisioning_profile.all(false).each do |profile|
```

Comment:
@KrauseFx I intended the default value to be "true", but the value for the download_all action to be false, since it speeds up download time a bit to skip invalid profiles. The HoundCI review didn't like it when I had the variable named, so I changed it to just the value. I can remove it entirely if that simplifies things.